scripts/import_from_ena_legacy.py

```diff
@@ -0,0 +1,849 @@
+"""
+This script reads from ENA FAANG data portal, parses and validates the data and stores into Elastic Search
+It is highly recommended to check out the corresponding rule set
+https://github.com/FAANG/faang-metadata/blob/master/rulesets/faang_experiments.metadata_rules.json
+to help understanding the code
+"""
+import click
+from constants import TECHNOLOGIES
+from elasticsearch import Elasticsearch
+from validate_experiment_record import *
+from validate_sample_record import *
+import constants
+from typing import Set, Dict, List
+from utils import determine_file_and_source, check_existsence, remove_underscore_from_end_prefix
+import re
+
+
+# in FAANG ruleset each technology has mandatory fields in the corresponding section, which is not expected in the
+# general ENA datasets, so only validate against Legacy standard
+RULESETS = ["FAANG Legacy Experiments"]
```

Comment:
RULESETS, STANDARDS, CATEGORIES, SPECIES_DICT, SPECIES_TAXOMONY_LIST, DATA_SOURCES, DATA_TYPES, ASSAY_TYPES_TO_BE_IMPORTED, EXPERIMENT_TARGETS, FIELD_LIST should be moved to constants

Reply:
Unfortunately I could not agree to simply put every single global variable into constants. 
A separate constants.py should only contain constants across the project, e.g. STAGE_ES_1 which is totally script independant.
Like RULESETS here, only applies to ena legacy, import biosamples could have its own RULESETS

Reply:
It's just a good coding practice to have all string constants in a separate place, as then it will be much much easier to find them (if we will have to change something). With current implementation we will have to search for this constants in separate files.

Reply:
For RULESETS you could write new data structure like this:
RULESETS = dict()
RULESETS['import_from_biosamples'] = [some data...]
RULESETS['import_from_ena'] = [some data...]
RULESETS['import_from_ena_legacy'] = [some data...]

Reply:
done in 054233d